src/please.go

```diff
@@ -456,6 +457,19 @@ var opts struct {
 var buildFunctions = map[string]func() int{
 	"build": func() int {
 		success, state := runBuild(opts.Build.Args.Targets, true, false, false)
+		if success {
+			if opts.Build.OutDir != "" {
+				for _, target := range state.ExpandOriginalLabels() {
+					outs := state.Graph.TargetOrDie(target).FullOutputs()
+					for _, out := range outs {
+						err := fs.CopyFile(out, opts.Build.OutDir+"/"+out, 777)
```

Comment:
Hmm, 777 isn't good. That would give everybody permissions to use this file, and make them all executable. We probably should stat the `out` to get it's existing permissions, using [os.Lstat()](https://pkg.go.dev/os#Lstat).